packages/block-editor/src/hooks/align.js

```diff
@@ -89,12 +89,21 @@ export function addAttribute( settings ) {
 		return settings;
 	}
 	if ( hasBlockSupport( settings, 'align' ) ) {
-		// Use Lodash's assign to gracefully handle if attributes are undefined
-		settings.attributes = assign( settings.attributes, {
-			align: {
-				type: 'string',
-			},
-		} );
+		const alignAttribute = {
+			type: 'string',
+		};
+
+		// Gracefully handle if settings.attributes is undefined.
+		if (
+			typeof settings.attributes === 'object' &&
+			settings.attributes !== null
```

Comment:
`typeof x === 'object'` returns `true` when `x` is `null`. There was a proposal to change this for ES6 several years ago, but it was rejected due to not wanting to break existing code. That's why I'm checking to make sure `settings.attributes` does not equal `null` in this if statement.

It might not actually be possible for `settings.attributes` to be `null`, so I might be able to remove the `null` check once we get stricter type checking.